variables.tf

```diff
@@ -467,7 +467,13 @@ variable "use_asm" {
 }
 
 variable "asm_role" {
-  description = "Specify arn of the ASM role (custom not created by the module)"
+  description = "Specify arn of the ASM role (custom not created by the module). Alias of boot_iam_role."
```

Comment:
Set a deprecation date and remove it in 2-3 months or so?

Reply:
Indeed. If terraform had a first class deprecation support I would have added that.
https://github.com/hashicorp/terraform/issues/18381

I'll add a note about deprecation though. I won't add a date though. It would just look unnecessarily silly if I miss the date...